apps/ledger-live-mobile/src/families/polkadot/Nominations/index.tsx

```diff
@@ -44,12 +49,18 @@ import EarnLight from "../../../images/illustration/Light/_003.png";
 import EarnDark from "../../../images/illustration/Dark/_003.png";
 
 type Props = {
-  account: Account;
+  account: AccountLike;
```

Comment:
No way to make it a `Polkadot` account and check in the parent component if the type is correct ?

Reply:
No